### DISCUSSION TITLE
v0.21.0-alpha.1

### DISCUSSION THREAD

**Discussion by atc0005** [atc0005/check-vmware#363] [Dev/Release Candidate]
## WARNING

**NOTE**: This is not considered a stable release.

## Changes since the last stable release

- `check_vmware_datastore` plugin
  - updated to emit additional performance data values (see references section below)
- other plugins
  - **not included in this release**

## References

- GH-341
- GH-362

<hr /><em>This discussion was created from the release <a href='https://github.com/atc0005/check-vmware/releases/tag/v0.21.0-alpha.1'>v0.21.0-alpha.1</a>.</em>

**Comment by atc0005** [comment]
@byolock,

If you have the time to test, your feedback would be welcome.

**Reply by Byolock** [reply]
This is what I get with this new version: 
![grafik](https://user-images.githubusercontent.com/25748003/136196388-d63b84b0-2ce8-4553-b3b9-a2da993b2c4d.png)
Looks good to me.

**Reply by atc0005** [reply]
Thanks for testing, I appreciate it.
